INFO     template_model.commons.variable_manager:variable_manager.py:40 [2025-09-24 18:25:41] 成功加载静态变量: VAR\uat_data.json
INFO     template_model.commons.variable_manager:variable_manager.py:64 [2025-09-24 18:25:41] 成功加载运行时变量: VAR\runtime_vars_cloud.json
INFO     template_model.conftest:conftest.py:74 [2025-09-24 18:25:41] 用例  开始登录，环境: uat
DEBUG    urllib3.connectionpool:connectionpool.py:1051 Starting new HTTPS connection (1): dev.lgcopytrade.top:443
DEBUG    urllib3.connectionpool:connectionpool.py:546 https://dev.lgcopytrade.top:443 "GET /api/sys/randomImage/1758709541818?_t=1758709541 HTTP/11" 200 None
INFO     template_model.conftest:conftest.py:143 第1次尝试 - 验证码识别结果: CJcG
DEBUG    urllib3.connectionpool:connectionpool.py:546 https://dev.lgcopytrade.top:443 "POST /api/sys/login HTTP/11" 200 None
INFO     template_model.conftest:conftest.py:186 uat环境登录成功（第1次尝试）
INFO     template_model.commons.variable_manager:variable_manager.py:111 [2025-09-24 18:25:41] 运行时变量已保存到: D:\Practice\QA-autoTest\template_model\VAR\runtime_vars_cloud.json
DEBUG    urllib3.connectionpool:connectionpool.py:546 https://dev.lgcopytrade.top:443 "PUT /api/blockchain/master-slave/editPa HTTP/11" 200 None
INFO     template_model.commons.api_base:api_base.py:339 [2025-09-24 18:25:44] PUT请求: /blockchain/master-slave/editPa | 数据: {'id': '1970043908604809217', 'direction': 'FORWARD', 'followingMode': 3, 'fixedProportion': None, 'fixedLots': 0.01}